creator-node/src/blacklistManager.js

```diff
@@ -76,6 +76,7 @@ async function _processBlacklist (ipfs, trackIdsToBlacklist, userIdsToBlacklist)
 
       // unpin from IPFS
       try {
+        // TODO: How should this be handled now that we no longer pin at all?
         await ipfs.pin.rm(CID)
```

Comment:
How should this be handled now that we no longer pin at all?

Reply:
@hareeshnagaraj we should check with @roneilr on this one. I guess the nuance I don't understand well enough here is if the gateway doesn't serve the content, but the ipfs node does, is that okay vs the ipfs node needs to clean it out as well. For the former, we would just have to delete the file from ipfs / not hydrate from the blacklist env var during start time, for the latter we do nothing since the middleware automatically rejects serving blacklisted files